content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
+author: bvdawson
+---
+
```

Comment:
What your thoughts on posting this as "CloudBees"?  

If I include a guest post introduction, should that be included in the body of the post as a lead in and Should it be in first-person?